apacheconfig/host.py

```diff
@@ -0,0 +1,22 @@
+import os
+
+
+class LocalHost(object):
+
+    def __init__(self):
+        self._environ = os.environ
+        self._path = os.path
+
+    @property
+    def environ(self):
+        return self._environ
+
+    @property
+    def path(self):
```

Comment:
Does it make sense to expose the whole `path` object as opposed to exposing just minimum necessary methods e.g. `isdir`, `listdir` etc.?

I am just thinking of building a URL or .zip archive reader within this 'reader' framework -- having to implement just the minimum required operations on files would be easier and clearer than exposing the `path` object with generally open-ended and partially irrelevant interfaces...

Reply:
I agree and will change this.